index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorgin="anonymous"></script>
+    <link rel="preconnect" href="https:// fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" <link rel="stylesheet"
+        href="https://fonts.googleleapis.com/css2?family=Material+Symbols+Rounded:ops,wght">
+    <title>MyPorfolio</title>
+</head>
+
+<body>
+    <div class="wraper">
+        <header>
+            <p>My Logo</p>
```

Comment:
- [ ] If you notice, most of the website has the logo which when clicked leads you to the home page. This means that the logo should be an `a` tag. I suggest that instead of a div, we adapt using `a` tag for [Semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) purposes and accessibility ☺️ .